packages/wonder-blocks-labeled-field/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@khanacademy/wonder-blocks-labeled-field",
+  "version": "0.0.0",
+  "design": "v1",
+  "description": "LabeledField handles accessibility and layout for associating labels with form components.",
+  "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "private": true,
+  "dependencies": {
+    "@babel/runtime": "^7.18.6",
+    "@khanacademy/wonder-blocks-color": "^2.0.1",
+    "@khanacademy/wonder-blocks-core": "^6.0.1",
+    "@khanacademy/wonder-blocks-layout": "^2.0.18",
+    "@khanacademy/wonder-blocks-spacing": "^4.0.1",
+    "@khanacademy/wonder-blocks-typography": "^2.1.4"
+  },
+  "peerDependencies": {
+    "aphrodite": "^1.2.5",
+    "react": "16.14.0"
+  },
+  "devDependencies": {
+    "wb-dev-build-settings": "^0.9.7"
```

Comment:
**issue:** This should use `@khanacademy/wb-dev-build-settings` to use the latest official version.